data/data/alibabacloud/cluster/vpc/eip.tf

```diff
@@ -0,0 +1,21 @@
+resource "alicloud_eip" "eips" {
+  count                = length(var.zone_ids)
+  description          = local.description
+  name                 = "${local.prefix}-eip-${count.index}"
```

Comment:
Update this field to be `address_name`. `name` is deprecated

Reply:
I observed that alicloud_eip has been deprecated from version 1.126.0, and recommend use new resource alicloud_eip_address, and the address_name is an attribute of alicloud_eip_address, so I replaced alicloud_eip with alicloud_eip_address.